packages/twenty-server/src/workspace/workspace-sync-metadata/commands/add-standard-id.command.ts

```diff
@@ -0,0 +1,157 @@
+})
+export class AddStandardIdCommand extends CommandRunner {
+  private readonly logger = new Logger(AddStandardIdCommand.name);
+
+  constructor(
+    @InjectDataSource('metadata')
+    private readonly metadataDataSource: DataSource,
+    private readonly standardObjectFactory: StandardObjectFactory,
+    private readonly standardFieldFactory: StandardFieldFactory,
+  ) {
+    super();
+  }
+
+  async run(): Promise<void> {
+    const queryRunner = this.metadataDataSource.createQueryRunner();
+
+    await queryRunner.connect();
+    await queryRunner.startTransaction();
+
+    const manager = queryRunner.manager;
+
+    this.logger.log('Adding standardId to metadata objects and fields');
+
+    try {
+      const standardObjectMetadataCollection =
+        this.standardObjectFactory.create(
+          standardObjectMetadataDefinitions,
+          {
+            // We don't need to provide the workspace id and data source id as we're only adding standardId
+            workspaceId: '',
+            dataSourceId: '',
+          },
+          {
+            IS_BLOCKLIST_ENABLED: true,
+            IS_CALENDAR_ENABLED: true,
+            IS_SELF_BILLING_ENABLED: true,
+          },
+        );
+      const standardFieldMetadataCollection = this.standardFieldFactory.create(
+        CustomObjectMetadata,
+        {
+          workspaceId: '',
+          dataSourceId: '',
+        },
+        {
+          IS_BLOCKLIST_ENABLED: true,
+          IS_CALENDAR_ENABLED: true,
+          IS_SELF_BILLING_ENABLED: true,
+        },
+      );
+
+      const objectMetadataRepository =
+        manager.getRepository(ObjectMetadataEntity);
+      const fieldMetadataRepository =
+        manager.getRepository(FieldMetadataEntity);
+
+      /**
+       * Update all object metadata with standard id
+       */
+      const updateObjectMetadataCollection: Partial<ObjectMetadataEntity>[] =
+        [];
+      const updateFieldMetadataCollection: Partial<FieldMetadataEntity>[] = [];
+      const originalObjectMetadataCollection =
+        await objectMetadataRepository.find({
+          where: {
+            fields: { isCustom: false },
+          },
+          relations: ['fields'],
+        });
+      const customObjectMetadataCollection =
+        originalObjectMetadataCollection.filter(
+          (metadata) => metadata.isCustom,
+        );
+      const standardObjectMetadataMap = new Map(
+        standardObjectMetadataCollection.map((metadata) => [
+          metadata.nameSingular,
+          metadata,
+        ]),
+      );
+
+      for (const originalObjectMetadata of originalObjectMetadataCollection) {
+        const standardObjectMetadata = standardObjectMetadataMap.get(
+          originalObjectMetadata.nameSingular,
+        );
+
+        if (!standardObjectMetadata && !originalObjectMetadata.isCustom) {
+          continue;
+        }
+
+        const computedStandardObjectMetadata = computeStandardObject(
```

Comment:
```suggestion
        const computedStandardObjectMetadata = computeStandardObject(
        computeStandardObject(
      standardObjectMetadata || {
        ...originalObjectMetadata,
        fields: standardFieldMetadataCollection,
      },
      originalObjectMetadata,
      customObjectMetadataCollection,
    );
```
nit: I feel like this is more readable like this